crates/terminal_view/src/terminal_panel.rs

```diff
@@ -727,6 +750,14 @@ impl TerminalPanel {
     fn serialize(&mut self, cx: &mut Context<Self>) {
         let height = self.height;
         let width = self.width;
+        let Some(database_id) = self
+            .workspace
+            .update(cx, |workspace, _| workspace.database_id())
```

Comment:
> `database_id`

This is `None` for ssh projects, but we want to serialize terminals in them, as it's working now.
In the database, there's also a `workspaces.session_id` id that we might use instead of this one.

Also, that `log_err` could be just `ok` as tries to update the weak pointer on the Workspace, that's only possible when there's a window/workspace alive.